### DISCUSSION TITLE
How do I deploy table transformer model weights locally to run the table parsing function offline?

### DISCUSSION THREAD

**Discussion by JudeZzz1997** [Filimoa/open-parse#44] [General]
When I use the table-transformers in order to parse the table as follows, I am prompted that I need to download the associated weights file from huggingface. 
However, my development environment does not allow me to download the weights directly from HF, so how do I deploy the downloaded weights file to run the program directly?Is there an interface parameter that enables the parser to read the local weights file?

```python
parser = openparse.DocumentParser(
    table_args={
        "parsing_algorithm": "table-transformers",
        "min_table_confidence": 0.8,
    }
)
parsed_doc2 = parser.parse(doc_with_tables_path)
```

**Comment by Filimoa** [comment]
Unfortunately this is pretty tightly coupled to hugging face - you'd need to consult their documentation. I don't have a ton of experience doing that.
